docs/hacking_howto.rst

```diff
@@ -90,6 +90,8 @@ These are required for the minimum installation:
 * libxslt and headers
 * libjpeg and headers
 * zlib and headers
+* libssl library
+* libmysqlclient library
```

Comment:
I have installed all the headers, but Open SSL was still needed. So addedd that.
And regarding the libmysqlclient library, MariaDB dont install it.
@willkg, can you please check with a fresh installation in Ubuntu?